Docker.md

```diff
@@ -0,0 +1,86 @@
+# Docker instructions
+
+1. Build the Docker image
+```sh
+docker build . -t airnode-alpha
+```
+
+2. Ensure that your `.env` file looks like [`/packages/deployer/.env.example`](/packages/deployer/.env.example) and is the current working directory.
+
+3. If you will be running [`deploy-first-time`](#deploy-first-time) or [`redeploy`](#redeploy), your `config.json` and `security.json` must be in the current working directory.
+
+4. Run the image with one of the following commands:
+
+### `deploy-first-time`
+```sh
+docker run -it --rm \
+  --env-file .env \
+  --env COMMAND=deploy-first-time \
+  -v $(pwd):/airnode/out \
```

Comment:
For the docker commands that require to read files from the current working directory (`config.json`, `security.json` or the receipt), we're creating a volume that maps the current working directory to `/airnode/out` in the container.